### ISSUE TITLE
Make an mdBook release?

### ISSUE THREAD

**Issue by projektir** [rust-lang-nursery/mdBook#503]
It seems that there wasn't a proper release since April, and the last release also lacked Windows versions as per https://github.com/rust-lang-nursery/mdBook/issues/481.

Could we get a new release/what's the process/timeline for releases? I would like to use one of the recent PRs about header partials in a project.

**Comment by Havvy** [comment]
There was a release two months ago, and it was tagged, but not put in https://github.com/rust-lang-nursery/mdBook/releases (along with 0.0.23, 0.0.24, and 0.0.25). So adding those as actual releases might be helpful?

But other than that, with header partials in now, it'd be a good time for another release.

**Comment by Michael-F-Bryan** [comment]
The configuration system has also completely changed, so technically we also need to bump the minor version. @Azerupi do you need to add me as an owner on crates.io for me to do a release?

**Comment by azerupi** [comment]
@Michael-F-Bryan Yes, I just added you and @budziq as crates.io authors. Let me know if it works.

**Comment by Michael-F-Bryan** [comment]
I'd like to get some feedback from the other maintainers, but if it's alright with everyone I was thinking of doing a release tomorrow when I've got time. 

From what I can think of, I'll need to:

- tag `master` with something like `v0.0.27` (or maybe `v0.1.0` because of the breaking configuration changes?)
- publish to crates.io (`cargo publish`)
- make a github release
- upload compiled binaries for linux, mac, and windows

cc: @budziq, @steveklabnik, @azerupi (because you've done this before for mdbook)

---

As a side note, it'd be nice if we could make travis generate releases automatically for us. For one, that would make sure we don't have a repeat of #481, plus it'd also automate away any tedious work accompanying a release. 

We could also probably use something like [cross] for generating the binaries.

[cross]: https://github.com/japaric/cross

**Comment by azerupi** [comment]
I used to do this all manually at first, but for the last couple of releases I used [cargo-release](https://github.com/sunng87/cargo-release) with great success.

As for the pre-built binaries, they used to be uploaded directly from Travis and AppVeyor when a new tag was added. AppVeyor failed to do this for the last releases. I think Travis still worked for Linux and MacOS builds. Anyways, with the changes in infrastructure, I am not sure any of them still works.
